symphony/app/fbcnms-projects/magmalte/app/components/DashboardKPIs.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2004-present Facebook. All Rights Reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import React from 'react';
+import EnodebKPIs from './EnodebKPIs';
+import GatewayKPIs from './GatewayKPIs';
+import Text from '../theme/design-system/Text';
+import {colors} from '../theme/default';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import Paper from '@material-ui/core/Paper';
+import {GpsFixed} from '@material-ui/icons';
+import {makeStyles} from '@material-ui/styles';
+
+const useStyles = makeStyles(theme => ({
+  cardTitle: {
+    marginBottom: theme.spacing(1),
+  },
+  cardTitleIcon: {
+    fill: colors.primary.comet,
+    marginRight: theme.spacing(1),
+  },
+  eventsTable: {
+    marginTop: theme.spacing(4),
+  },
+}));
+
+export default function() {
+  const classes = useStyles();
+
+  return (
+    <>
+      {/* TODO: Can come back and make this a reusable component for other cards */}
```

Comment:
why not do it now? :)

Reply:
hmm you should be able to pass in a component via props like normal, I believe we do this in things like `LteSections` where we pass in components for rendering icons or other things that should be rendered... or anything in the code that does `component={NestedRouterLink}`